test/Datadog.Trace.ClrProfiler.Managed.Tests/IntegrationSignatureTests.cs

```diff
@@ -1,37 +1,74 @@
 using System.Collections.Generic;
-using System.Diagnostics.CodeAnalysis;
 using System.Linq;
 using System.Reflection;
+using Datadog.Trace.ClrProfiler.Integrations;
 using Xunit;
 
 namespace Datadog.Trace.ClrProfiler.Managed.Tests
 {
     public class IntegrationSignatureTests
     {
-        [SuppressMessage("StyleCop.CSharp.SpacingRules", "SA1009:ClosingParenthesisMustBeSpacedCorrectly", Justification = "Reviewed.")]
-        public static IEnumerable<object[]> GetWrapperMethods()
+        private static readonly List<MethodInfo> StaticInstrumentations = new List<MethodInfo>()
```

Comment:
I feel like you could get the static methods dynamically so that, if we add new static wrapper methods, we don't have to remember to add them here.

Reply:
I think I spoke too soon without realizing you're trying to limit this to just the AspNetCoreMvc2Integration. It may still be beneficial to only have to list the class names and then we'd dynamically get all of the (publicly accessible) integration methods from that class.

Reply:
All of our methods are static, regardless of whether the target is. 
I think we would need to decorate whether the target is static in the attribute.

Reply:
At the very least, the arg counts need to match in the profiler for the instrumentation to happen, and if it's a static method, then this test will automatically fail until it's added to this list.